permits/templates/permits/permit_request_new.html

```diff
@@ -1,8 +1,12 @@
 {% extends "base_generic.html" %}
 {% load bootstrap4 %}
 {% load i18n %}
+{% block navbar %}
+{% load permit_progressbar %}
+{% endblock navbar %}
 
 {% block content %}
+{% permit_progressbar 1 %}
```

Comment:
You'll want to use `{% permit_progressbar permit_request %}` to use the current permit request.